### PR TITLE
Change label "username" to "Email" in login form

### PR DESCRIPTION
**Pull request by Arsnael**
That has been in some feedbacks that it was confusing.

I also saw that the text input type for the url in the login form was emailAddress, which is wrong (I think), so I changed it to url type.

email-label

### CODE REVIEW

File: lib/l10n/intl_en.arb [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "placeholders_order": [],
     "placeholders": {}
   },
-  "username": "username",
+  "username": "Email",
```
Comment (hoangdat) [inline lib/l10n/intl_en.arb:21]: why you not change the variable name `username` to `email` too?

File: lib/main/localizations/app_localizations.dart [hunk 2]
```diff
@@ -35,7 +35,7 @@ class AppLocalizations {
   }
 
   String get username {
```
Comment (hoangdat) [inline lib/main/localizations/app_localizations.dart:37]: change this field to `email`

File: lib/l10n/intl_en.arb (hunk 1 above)
Comment (Arsnael) [reply lib/l10n/intl_en.arb:21]: Thought about it, but wasn't sure about all the links in the code... But it would make sense I agree, will give it a shot

**Review by hoangdat — APPROVED** [review]

**Comment by Arsnael** [comment]
Squashed the fixup

**Pull request merged**